src/js/tech/html5.js

```diff
@@ -273,13 +273,26 @@ class Html5 extends Tech {
       return;
     }
     const listeners = {
-      change(e) {
-        techTracks.trigger({
+      change: (e) => {
+        const event = {
           type: 'change',
           target: techTracks,
           currentTarget: techTracks,
           srcElement: techTracks
-        });
+        };
+
+        techTracks.trigger(event);
+
+        // if we are a text track change event, we should also notify the
```

Comment:
May be worth adding a note about the alternative approach in case there's ever time to implement it, if it's still seen as a benefit.

Reply:
The comment is really long. Maybe I'll short it to with a link here and elaborate a bit more here.